addon/helpers/moment-same.js

```diff
@@ -1,5 +1,4 @@
 import Ember from 'ember';
-import moment from 'moment';
 
 export function momentSame([date1, date2, unit]/*, hash*/) {
   if (!date1 || !date2) {
```

Comment:
can we check if this guard is necessary? I don't think it is. Also - this helper can replace `moment-same-month` and `moment-same-decade`, @cantsin can you take a look at this? FYI @taras